extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -263,8 +267,10 @@ export const AssetDetail = ({
               {t("No transactions to show")}
             </div>
           )}
-        </SimpleBarWrapper>
-      </div>
+          {/* </SimpleBarWrapper> */}
+        </div>
+      </View.Content>
+      {/* TODO: fix the slideup modal */}
```

Comment:
We should create a "slide over" modal component. I haven't updated it yet, but it should look OK on smaller screen sizes. The width will stretch across the whole screen, which is not great for larger screen sizes.